.changeset/four-oranges-clap.md

```diff
@@ -0,0 +1,6 @@
+---
+"@clerk/clerk-js": minor
+"@clerk/types": minor
+---
+
+Trigger UserVerification within UserProfile
```

Comment:
Let's rewrite the changelog to specify for which sections (e.g. 2FA) this new opt-in behavior can be used